src/views/List.jsx

```diff
@@ -80,76 +49,31 @@ export function List({ data, listToken }) {
 					);
 				})}
 		</div>
+	) : (
+		<div>
+			{listToken ? (
+				<>
+					<p>
+						A shopping list with token name: "{listToken}" has been created and
+						is currently empty.
+					</p>
+					<Link style={{ color: 'white' }} to="/add-item">
```

Comment:
Curious to know why you all added this inline style in a few places. It makes the links invisible on my browser: 
![image](https://user-images.githubusercontent.com/58532369/203185775-b7f58194-4d81-4ea2-82fe-4450caa5dfeb.png)

Reply:
Oops😱, thank you for pointing that out. Changed color💚